### ISSUE TITLE
Use WCAG AA or AAA standards for colour contrast

### ISSUE THREAD

**Issue by abhidg** [globaldothealth/list#2796]
Color contrast is an important issue in accessibility. WCAG (https://www.w3.org/WAI/standards-guidelines/wcag/) recommends AA for general use, with AAA being the highest contrast. We should have at least AA contrast everywhere, with AAA where possible and where it is in line with our visual design guidelines.

Here's the detailed WCAG guidelines for color contrast https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

Some apps that help with calculation of contrast

https://usecontrast.com/ (macOS)
https://coolors.co/contrast-checker/ (web)
https://apps.kde.org/en-gb/kontrast/ (Linux, KDE)
https://apps.gnome.org/en-GB/app/org.gnome.design.Contrast/ (Linux, GNOME)
